irohad/ametsuchi/impl/mutable_storage_impl.cpp

```diff
@@ -52,6 +52,7 @@ namespace iroha {
             function) {
       auto execute_transaction = [this](auto &transaction) {
         command_executor_->setCreatorAccountId(transaction.creatorAccountId());
+        command_executor_->setIsGenesis(true);
```

Comment:
Does it mean that all transactions are executed as genesis block?

Reply:
Yes, since mutable storage doesn't have CommandValidator, this flag is set here every time.

Reply:
Please rename to something like doValidation()